src/medCore/gui/viewContainers/medViewContainer.cpp

```diff
@@ -636,35 +636,38 @@ void medViewContainer::dropEvent(QDropEvent *event)
 {
     const QMimeData *mimeData = event->mimeData();
     medDataIndex index = medDataIndex::readMimeData(mimeData);
-    if(!index.isValidForSeries())
-        return;
 
-    if(d->userSplittable)
+    // User can drop a study or a series into the view
+    if(index.isValidForSeries() || index.isValidForStudy())
```

Comment:
i just added the `isValidForStudy` condition. The other changes on this function are indentation